atsign-foundation/atmosphere_pro
# UI Redesign - Mobile - Trusted Senders fixes #222

**- What I did**
Created a new UI design for the trusted senders screen and a remove trusted sender dialog which closes #222 

**- How I did it**
I created a new screen for the trusted senders screen  inside the trusted contacts folder

**- How to verify it**
add the code to the welcome screen and you should be able to click the button to view the new screen
```
 CustomOutlinedButton(
     buttonText: 'Trusted Senders',
      onPressed: () {
              showModalBottomSheet(
                  context: context,
                  elevation: 10,
                  isScrollControlled: true,
                  //color with gradient
                  barrierColor: Colors.white
                      .withOpacity(0.25)
                      .withOpacity(0.6),
                  shape: RoundedRectangleBorder(
                      borderRadius: BorderRadius.only(
                          topLeft:
                              Radius.circular(20.toWidth),
                          topRight:
                              Radius.circular(20.toWidth))),
                  builder: (context) => BackdropFilter(
                      filter: ImageFilter.blur(
                        sigmaY: -2,
                      ),
                      child: TrustedSenders()));
               },
    ),
```
**- Description for the changelog**
Added trusted sender screen and dialog

![Simulator Screen Shot - iPhone 11 Pro Max - 2022-10-03 at 15 57 57](https://user-images.githubusercontent.com/47469638/193617809-a09a334e-9a2c-4585-9103-0b0d811ded09.png)
![Simulator Screen Shot - iPhone 11 Pro Max - 2022-10-03 at 15 57 54](https://user-images.githubusercontent.com/47469638/193617829-24e40949-855a-4aa2-867a-727eff5cd1cd.png)
